corehq/apps/reports/display.py

```diff
@@ -88,39 +88,49 @@ def __init__(self, domain, xmlns, app_id):
     def get_label(self, lang=None, separator=None):
         if separator is None:
             separator = " > "
+
+        return (
```

Comment:
[?w=1](https://github.com/dimagi/commcare-hq/pull/31923/commits/666e0e00e6b68e89bde393910dd2cfa755eba2fc?w=1) reads a lot easier for this commit